stories/assembly-gcp-monitoring-logging.adoc

```diff
@@ -5,6 +5,11 @@ ifdef::context[:parent-context: {context}]
 
 :context: gcp-monitor-log
 
+[NOTE]
+====
+This feature will be removed in a future release.
```

Comment:
You should also add something to the Release  notes appendix.

Reply:
Should I add a deprecation note under `Deprecated and removed features` section in[ release notes](https://access.redhat.com/documentation/en-us/ansible_on_clouds/2.x/html-single/red_hat_ansible_automation_platform_from_gcp_marketplace_guide/index#assembly-appendix-aap-release-notes)?